qiskit_nature/second_q/operators/commutators.py

```diff
@@ -45,7 +45,7 @@ def commutator(op_a: SparseLabelOp, op_b: SparseLabelOp) -> SparseLabelOp:
     Returns:
         The computed commutator.
```

Comment:
I think what you said in the description: "Basically, this means that the user needs to normal-order the result themselves if needed (which is totally fine)" should appear in the docsctring of the `commutator`/`anti_commutator`. The user might be confused otherwise